### PR TITLE
DM-11390: Plug prototype pipeline script into verify_ap framework

### PR DESCRIPTION
**Pull request by mrawls**

### CODE REVIEW

File: index.rst [hunk 1]
```diff
@@ -27,21 +27,25 @@ summarized below and in :ref:`Figure 1<fig-nightly-proc>`, the first portion of
...
 
 A number of packages in the LSST Software Stack have been developed to perform
 portions of one or more of these four tasks, but they are not yet linked together
 in an automated way. This Prototype AP Pipeline is the first step toward the goal
 of an automatic Level 1 Pipeline that can process a set of raw images through 
 each of these steps and verify that the output is what we expect.
 
+To achieve this, there are two main python modules under development: 
+`ap_pipe <https://github.com/lsst-dm/ap_pipe>`_ and `ap_verify <https://github.com/lsst-dm/ap_verify>`_.
+The former is described here, and is used by the latter.
```
Comment (kfindeisen) [inline index.rst]: Maybe give a bit more detail, e.g. that `ap_pipe` is responsible for running the pipeline while `ap_verify` verifies the output?

File: index.rst [hunk 2]
```diff
@@ -27,21 +27,25 @@ summarized below and in :ref:`Figure 1<fig-nightly-proc>`, the first portion of
...
+
 
 .. _Dataset:
 
 Dataset
 =======
 
-We use images from the High Cadence Transient Survey (HiTS, [for16]_), a three-year survey 
+We use images from the High Cadence Transient Survey (HiTS) :cite:`for16`, a three-year survey
```
Comment (kfindeisen) [inline index.rst:49]: Did the technote powers that be say anything about moving references into central bibliography files? If you're going to have to do that later, than an ADS ID might be a more portable name than `for16` (but beware `...` in IDs).

File: index.rst [hunk 3]
```diff
@@ -52,163 +56,110 @@ The distribution of the HiTS survey fields in the sky is shown in :ref:`Figure 2
...
 
-To streamline this process, we wrote a python script called ``decam_process.py`` to handle each step.
-It is `available on GitHub <https://github.com/lsst-dm/decam_hits/blob/master/decam_process.py>`_.
-While it was only used on the :ref:`Dataset <Dataset>` described above, it can
-in principle be applied to any set of raw DECam images using the LSST Stack, as shown below.
+The prerequisites for running ``ap_pipe`` are:
 
+- The `LSST Science Pipelines <https://pipelines.lsst.io/install/lsstsw.html>`_ 
+  stack with the ``lsst_apps`` modules (a subset of ``lsst_distrib``)
+- The ``obs_decam`` and ``ap_pipe`` modules
```
Comment (kfindeisen) [inline index.rst]: Packages, not modules.

File: index.rst [hunk 4]
```diff
@@ -52,163 +56,110 @@ The distribution of the HiTS survey fields in the sky is shown in :ref:`Figure 2
...
-Be aware that the difference imaging will not work well in this case, however, because the visits 
-will not be precisely lined up to the same WCS.
 
-Once you are ready, run the following:
+   setup lsst_apps
+   git clone https://github.com/lsst/obs_decam.git
+   git clone https://github.com/lsst-dm/ap_pipe.git
+   setup -k -r obs_decam
+   setup -k -r ap_pipe
+   git clone https://github.com/lsst/ap_verify_hits2015.git
```
Comment (kfindeisen) [inline index.rst:110]: Is something like this really inside the scope of the tech note? I would think the target audience already has their own way of setting up the Stack.

File: index.rst [hunk 5]
```diff
@@ -52,163 +56,110 @@ The distribution of the HiTS survey fields in the sky is shown in :ref:`Figure 2
...
-   image and visit 411033 as the science image. CCDs 2 and 61 are nonoperational, and
-   a portion of CCD 31 is also not working. The other CCDs all perform as expected.
-
+   Calibrated science exposure (top) and difference image (bottom) for CCD 25 of HiTS field ``Blind15A_40``.
+   This example uses visit 410929 as the template image and visit 410985 as the science image.
 
+Future work will use coadded 2014 visits of the three HiTS fields described in Dataset 
+as templates and the corresponding 2015 visits as science.
+This pipeline is a core component of the 
+`AP Minimum Viable System <https://confluence.lsstcorp.org/display/~ebellm/AP+Minimum+Viable+System>`_
```
Comment (kfindeisen) [inline index.rst]: This is a non-public link; are you ok with that? If so, it might still be worth mentioning.

**Review by kfindeisen — APPROVED** [review]
Looks good. A few questions/comments.

File: index.rst (hunk 2 above)
Comment (mrawls) [reply index.rst:49]: The short answer is, I don't believe there is a standard, documented way for handling this yet. Good point to keep in mind once there is, though!

**Pull request merged**